lib/periodic_jobs.rb

```diff
@@ -224,10 +224,10 @@
   # Every 15min job that sends missing Pega statuses to DataDog
   mgr.register('*/15 * * * *', 'IvcChampva::MissingFormStatusJob')
 
-  # Engine version: Hourly jobs that update DR SavedClaims with delete_date
-  mgr.register('10 * * * *', 'DecisionReviews::HlrStatusUpdaterJob')
-  mgr.register('15 * * * *', 'DecisionReviews::NodStatusUpdaterJob')
-  mgr.register('40 * * * *', 'DecisionReviews::ScStatusUpdaterJob')
+  # Engine version: Sync non-final DR SavedClaims to LH status
+  mgr.register('10 0,4,8,12,16,20 * * *', 'DecisionReviews::HlrStatusUpdaterJob')
```

Comment:
I think you can write these as :

`10 */4 * * *`
`15 1-23/4 * * *`
`30 2-22/4 * * *`